tchannel/serializer/json.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+import json
+
+from tchannel.schemes import JSON
+
+
+class JsonSerializer(object):
+    def __init__(self):
```

Comment:
I mean, there's no reason they can't be grouped together in a `Serializer` class. Instances of that get passed around, and consumers don't care how they were instantiated or whether they carry state. Like, if I wanted to implement a  generic serializer that wraps another serializer in an envelope of some kind, the top-level function approach would leave me unable to do that without serious refactoring.